mrp_food/models/product_supplierinfo.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+from odoo.addons import decimal_precision as dp
+
+
+class SupplierInfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    product_standard_price = fields.Float(
+        string="Product actual standard price",
+        compute="_compute_product_standard_price",
+    )
+
+    product_tmpl_code = fields.Char(related="product_tmpl_id.default_code")
+
+    product_tmpl_name = fields.Char(related="product_tmpl_id.name")
+
+    # Should be in a specific glue module i think
+    supplierinfo_standard_price_w_discount = fields.Float(
```

Comment:
alors le "w" c'est pas top standard, et en plus, ça peut-être la diminutif de "with" et de "without". (bon même si pour without, c'est wo parfois...

```suggestion
    theoritical_standard_price = fields.Float(
```

Reply:
ah yes merci :ok_hand:

Reply:
fait :heavy_check_mark: